js/src/annotations/osd-svg-overlay.js

```diff
@@ -339,8 +352,7 @@
         //should remove the styles added for newly created annotation
         for(var i=0;i<_this.draftPaths.length;i++){
           if(_this.draftPaths[i].data && _this.draftPaths[i].data.newlyCreated){
-            _this.draftPaths[i].strokeWidth /= _this.draftPaths[i].data.newlyCreatedStrokeFactor;
-            _this.draftPaths[i].data.currentStrokeValue /= _this.draftPaths[i].data.newlyCreatedStrokeFactor;
```

Comment:
How is it that in a number of places you are able to remove the strokeValue? Is it just not used in these instances?